README.md

```diff
@@ -354,6 +354,22 @@ Allows you to add specific actions for notification with specific category.
 
 ---
 
+### `removePendingNotificationRequests()`
```

Comment:
Hi,
There is a small typo between the title and the exemple in code.
`removePendingNotificationRequests` vs `removeDeliveredNotifications`
😉